stripe/src/main/java/com/stripe/android/view/CountryTextInputLayout.kt

```diff
@@ -148,6 +150,36 @@ internal class CountryTextInputLayout @JvmOverloads constructor(
         }
     }
 
+    override fun onSaveInstanceState(): Parcelable? {
+        selectedCountry?.let {
+            return CountryTextInputLayoutState(
+                it.code.value,
+                super.onSaveInstanceState()
+            )
+        } ?: run {
+            return super.onSaveInstanceState()
+        }
+    }
+
+    override fun onRestoreInstanceState(state: Parcelable?) {
+        if (state is CountryTextInputLayoutState) {
+            super.onRestoreInstanceState(state.superState)
+            CountryCode.create(state.countryCode).let { countryCode ->
+                updatedSelectedCountryCode(countryCode)
+                updateUiForCountryEntered(countryCode)
+                requestLayout()
+            }
```

Comment:
Can you move this into a `restoreSelectedCountry` method and call it from your tests? Seems like this is basically what you're doing already.